eng/tools/TypeSpecValidation/src/rules/format.ts

```diff
@@ -8,9 +8,9 @@ export class FormatRule implements Rule {
 
   async execute(folder: string): Promise<RuleResult> {
     // Format parent folder to include shared files
-    let [stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
 
-    let success = errorOutput == null ? false : true;
+    let [err, stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
+    let success = err || errorOutput ? false : true;
```

Comment:
I believe we should only treat `errorOutput` as implying "Error" if we know a tool is buggy, like `tsp format`.  Add a comment explaining this with a link to the bug you opened against `tsp format`.